`msftidy` is complaining about:

```
modules/exploits/multi/misc/claymore_dual_miner_remote_manager_rce.rb:172 - [ERROR] fail_with requires a valid Failure:: reason as first parameter: fail_with(Exploit::Failure::NoTarget, 'No matching target') 
```

`Exploit::Failure::NoTarget` should probably be `Failure::NoTarget`

--- a/modules/exploits/multi/misc/claymore_dual_miner_remote_manager_rce.rb
+++ b/modules/exploits/multi/misc/claymore_dual_miner_remote_manager_rce.rb
@@ -169,7 +169,7 @@ class MetasploitModule < Msf::Exploit::Remote
   def exploit
     target = select_target
     if target.nil?
-      fail_with(Exploit::Failure::NoTarget, 'No matching target')
+      fail_with(Failure::NoTarget, 'No matching target')
     end
     if (target['Platform'].eql?('linux') && payload_instance.name !~ /linux/i) ||
       (target['Platform'].eql?('windows') && payload_instance.name !~ /windows/i)

--- a/modules/exploits/multi/misc/claymore_dual_miner_remote_manager_rce.rb
+++ b/modules/exploits/multi/misc/claymore_dual_miner_remote_manager_rce.rb
@@ -77,10 +77,10 @@ class MetasploitModule < Msf::Exploit::Remote
     tmp << buf
     tmp2 = tmp.string
     hex = ''
-    if tmp2.scan(/\w+/)[5]
-      return self.targets[1]
-    elsif tmp2.scan(/\w+/)[7]
+    if tmp2.scan(/\w+/)[7]
       return self.targets[2]
+    elsif tmp2.scan(/\w+/)[5]
+      return self.targets[1]
     else
       return nil
     end

